package.json

```diff
@@ -55,9 +55,9 @@
     "@commitlint/cli": "^13.1.0",
     "@commitlint/config-conventional": "^13.1.0",
     "@octokit/core": "^3.5.1",
-    "@opentelemetry/instrumentation-bunyan": "~0.24.0",
-    "@opentelemetry/instrumentation-http": "~0.23.0",
-    "@opentelemetry/instrumentation-pino": "~0.23.0",
+    "@opentelemetry/instrumentation-bunyan": "^0.24.0",
+    "@opentelemetry/instrumentation-http": "^0.24.0",
+    "@opentelemetry/instrumentation-pino": "^0.24.0",
```

Comment:
Why did we change `~` to `^` ?

Reply:
I highly suggest we use `~` for packages that are pre 1.0 and use `^` only for stable packages

Reply:
You're right. Changed those.